### ISSUE TITLE
Tests no longer run in version 6.8.15

### ISSUE THREAD

**Issue by d2a-raudenaerde** [cbeust/testng#594]
Hi all, 

I saw the new release coming, and jenkins decided to no longer run our tests. We use test-ng with profiles and relative paths, so there might be an issue in our config, or a bug. 

This is our relevant maven confing:

``` xml
        <profile>
            <id>fast</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-surefire-plugin</artifactId>
                        <version>2.14.1</version>
                        <configuration>
                            <!-- https://community.jboss.org/message/647753 -->
                            <argLine>${argLine} -Xms1024m -Xmx4096m -XX:MaxPermSize=750m</argLine>
                            <enableAssertions>false</enableAssertions>
                            <suiteXmlFiles>
                                <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
                            </suiteXmlFiles>

                        </configuration>
                    </plugin>
                </plugins>
            </build>
        </profile>
```

The testng.xml file:

``` xml
<suite name="Suite1" verbose="1">
    <test name="AllTests">
        <groups>
            <run>
                <exclude name="slowtests"></exclude>
                <exclude name="only-on-jenkins"></exclude>
            </run>
        </groups>
        <packages>
            <package name="my.package.*" />
        </packages>
    </test>
</suite>
```

It used to run all tests that were not excluded, but now it's running nothing anymore.

**Comment by VladRassokhin** [comment]
May be caused by #593 use 6.8.14 for now

**Comment by cbeust** [comment]
I'm deploying a fix to Maven right now, but it would be nice if you could
confirm it's indeed that issue that's causing your tests to stop running.

-- 
Cédric

**Comment by sskuts** [comment]
I can confirm the same issue. When I updated ReportNG library with new TestNG version, test execution results are just: 0 passed / 0 failed / 0 skipped.

I've just posted comments here: https://groups.google.com/forum/#!topic/testng-users/jBpNtx0n3xE

**Comment by sskuts** [comment]
I can also confirm that 6.8.14 works fine. Does it contain OutOfMemoryExceptions fix? If so, I'll switch to it, util 6.8.15 version issue is fixed.

**Comment by sskuts** [comment]
Fixed for me in 6.8.17 version. Thanks!

**Comment by krmahadevan** [comment]
Closing this issue based on the above comment.
